### ISSUE TITLE
Implement v2 `Acknowledgement` rpc handler

### ISSUE THREAD

**Issue by damiannolan** [cosmos/ibc-go#7356]
Requires v2 `IBCModule` design and clarity regarding new acknowledgement type and commitment structure

**Comment by bznein** [comment]
> done in [#7452](https://github.com/cosmos/ibc-go/pull/7452) it seems? Re-open if mistaken!

Yeah I keep forgetting that "closes" keyword doesn't work for PRs not targeting main

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
